crates/sui-core/src/authority.rs

```diff
@@ -791,6 +791,15 @@ impl AuthorityState {
         self.database.get_object(object_id)
     }
 
+    pub async fn get_framework_object_ref(&self) -> SuiResult<ObjectRef> {
+        // unwrap safe because framework object should always exist.
```

Comment:
I would have expected the "should" comment to be in an `expect` where you're using an `unwrap`, below.